.github/copilot-instructions.md

```diff
@@ -0,0 +1,103 @@
+---
+author: wadepickett
+ms.author: wpickett
+ms.date: 07-29-2025
+---
+
+# Copilot Instructions for `dotnet/AspNetCore.Docs`
+
+## Introduction
+This document contains general and repository-specific instructions for GitHub Copilot when assisting with the `dotnet/AspNetCore.Docs` repository. **Unless otherwise specified, all ".NET" references refer to modern .NET, not .NET Framework.**
+
+## General Guidelines
+
+### Issue Handling
+When creating a PR for an issue:
+1. Read the full issue and all linked references
+2. Study code samples from linked PRs for pre-release features
+3. For labeled issues:
+ - **new-feature:** State which version introduced the feature
+ - **bug:** Focus on correcting technical inaccuracies
+4. When you're assigned an issue, after you've completed your work and the workflows (status checks) have run, check to make sure there are no build warnings under the OpenPublishing.Build status check. If there are, open the build report (under View Details) and resolve any build warnings you introduced.
+
+### Markdown File Naming and Organization
+- If you're adding a new Markdown file, it should be named in all lowercase with hyphens separating words. Also, omit any filler words such as "the" or "a" from the file name.
+
+### API References and Verification
+- Never guess API documentation IDs, always verify. If unable to verify, say so.
+- Check if features were introduced after .NET Core 3.1
+- Default to most compatible syntax when uncertain
+
+### Links and References
+- Use relative links for files within this repo
+- When adding external links into a document, strip out the language/culture segment of the URL if present (example: remove the `en-us` or `en` part of links to English versions of webpages)
```

Comment:
I've never seen -en-by itself; not needed in this example

``suggestion
- When adding external links into a document, strip out the language/culture segment of the URL if present (example: remove the `en-us` part of links to English versions of webpages)
```

Reply:
I've seen it several times over the years. The one that jumps to mind is Wikipedia ...

https://en.wikipedia.org/wiki/.net

... and ...

https://wikipedia.org/wiki/.net

... sets the host to English, given our culture.